### DISCUSSION TITLE
preeclass phyton -- jupyter

### DISCUSSION THREAD

**Discussion by Daizane** [teamalgoritma/community#619] [Q&A] [answered]
gan, di video "pree-class phyton" saya mentok di menit 35, bagian jupyter buka di incognito hasilnya ndak bisa di buka di incognito, tapi di google chorome  biasanya bisa 
![image](https://user-images.githubusercontent.com/128264111/231033419-bc212772-7ace-4ed4-987c-1a9a04b7b222.png)

**Comment by Daizane** [comment]
![image](https://user-images.githubusercontent.com/128264111/231036361-9173c517-37a6-466f-b3d8-f69c8e0a9e97.png)
lanjutan dari foto sebelum nya cuman muncul respons seperti ini

**Comment by Daizane** [comment]
![Uploading image.png…]()

**Comment by Daizane** [comment]
file nya ndak bisa buka gan

**Comment by Daizane** [comment]
![image](https://user-images.githubusercontent.com/128264111/231038212-07fe3ed0-2a16-4bd1-8815-bbc849369907.png)

**Comment by amaliasf** [comment, marked as answer]
Halo Pak @Daizane , bisa dipastikan ketika menjalankan dan membuka link jupyter notebook, Aplikasi Command Prompt masih menunjukkan bahwa status **jupyter notebook** masih running. Berdasarkan screenshot yang Bapak lampirkan, adanyat interupsi yang terjadi sehingga statusnya menjadi shutdown.

Agar lebih mudah menjalankan jupyter notebook, Bapak bisa membuka via **Anaconda Prompt**. Lalu jalankan `jupyter notebook`.

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

**Reply by Daizane** [reply]
ouh iya lupa saya, soalnya saya ada sleepkan komputer 24 jam, makasih ya

**Comment by Daizane** [comment]
siang gan, masih dengan meteri pembelajaran yang sama pandas saya yang awalnya versi 1.5.3, sudah berhasil update ke 2.0.0, tetapi tadi barusan chek lagi versi pandasnya sepertinya gagal dan menjadi versi 1.5.3 kembali, sudah instal kembali anaconda promp nnya menggunakan "pip install pandas==2.0.0" dan "pip install pandas --upgrade" .. apakah ada yang ketinggalan ya ?. koneksi semua sudah oke, dan anaconda promp sudah aktif jg
![image](https://user-images.githubusercontent.com/128264111/233229088-e8d2d086-9faf-4c42-9e00-9a18cd89f4c0.png)

**Reply by Daizane** [reply]
sebelummnya ada keluar notifikasi jentela kecil error gitu saya close muncul lagi berualng2x tapi sudah tidak muncul algi, cuman tadi saya ndak sempat screanshot isinya

**Reply by Daizane** [reply]
![image](https://user-images.githubusercontent.com/128264111/233246140-ac99837c-6c02-45ab-93ad-f70411c95bac.png)

**Reply by Daizane** [reply]
![image](https://user-images.githubusercontent.com/128264111/233247635-3aa075ba-1722-4b1a-a865-a7726d271ac4.png)

**Reply by Daizane** [reply]
ini kembali lagi  seperti gini ndak bisa, kadang 2.0.0 kadang 1. sekian.

ujung ujung nya materi csv nya ndak bisa di buka di jupyter
